src/libmongoc/src/mongoc/mongoc-stream-tls-openssl.c

```diff
@@ -543,6 +543,28 @@ _mongoc_stream_tls_openssl_check_closed (mongoc_stream_t *stream) /* IN */
 }
 
 
+static bool
+_mongoc_stream_tls_openssl_check_cert_failure (SSL *ssl, bson_error_t *error)
```

Comment:
Minor nit: I wonder if the name `_mongoc_stream_tls_openssl_cert_verify_failed` might be more consistent with the other functions that return `bool` based on the state of a parameter.